source/agora/crypto/Schnorr.d

```diff
@@ -121,28 +104,49 @@ public Point extractNonce (Signature sig) @safe @nogc nothrow pure
 
```

Comment:
The documentation is now outdated (it talks about binary blobs).

Reply:
Updated.